### PR TITLE
BL-3035 Talking Book Not starting up properly in a brand-new book

### PR DESCRIPTION
**Pull request by JohnThomson**
[Review on Reviewable](https://reviewable.io/reviews/bloombooks/bloomdesktop/922)

### CODE REVIEW

File: src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts [hunk 1]
```diff
@@ -382,6 +382,11 @@ function showToolboxChanged(showing: boolean): void {
     if (currentTool) {
         if (showing) currentTool.hideTool();
         else currentTool.showTool();
+    } else {
```
Comment (hatton) [inline src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts]: This seems fragile; should toolbox really know about any individual tool?

Comment (JohnThomson) [inline src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts]: I think I found a better approach: just activate the first tool

Comment (hatton) [inline src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts]: OK. We will probably want to show the previously shown tool instead, but that doesn't need to be part of this.

Comment (JohnThomson) [inline src/BloomBrowserUI/bookEdit/toolbox/toolbox.ts]: It should already show the most recently shown tool for the current book if there is one; this change is only for NEW books. Do we want those to remember which tool was active in the previously selected book? But anything but talking book may not even be turned on for a new book...

**Pull request merged**